nipype/interfaces/ants/registration.py

```diff
@@ -321,6 +321,8 @@ class RegistrationInputSpec(ANTSCommandInputSpec):
     winsorize_lower_quantile = traits.Range(
         low=0.0, high=1.0, value=0.0, argstr='%s', usedefault=True, desc="The Lower quantile to clip image ranges")
 
+    float_computations = traits.Int(argstr='--float %d', value=0, desc="Use single floating point for computations")
```

Comment:
for example as used here:

https://github.com/nipy/nipype/blob/master/examples/rsfmri_vol_surface_preprocessing_nipy.py#L506